desktop/cmp/grid/GridModel.js

```diff
@@ -228,6 +232,11 @@ export class GridModel {
         return this.selModel.singleRecord;
     }
 
+    @computed
+    get hasSubFields() {
+        return !!this.subFields.length;
```

Comment:
'subFields' are not observable (and in fact not even declared at top of class).  This @computed does nothing without that.  Make subFields observable.ref if so desired, but perhaps they really could be part of the "immutable" public properties.  Do we expect them to change?